src/utils/aws.ts

```diff
@@ -19,3 +20,19 @@ export const ensureCredentials = async (): Promise<void> => {
     store.AWS.Credentials.secretAccessKey = answers['Secret Key'];
   }
 };
+
+let credentialsSet = false;
+/**
+ * This may be called multiple times throughout the app
+ * so we must ensure we only update AWS config once.
+ */
+export function setCredentials(): void {
```

Comment:
And this uses the store so it causes the store to load the credentials from disk. IF the user hasn't gone through configure, it will no longer prompt the user for credentials. It will now just error out.

Reply:
I can't figure out how to prompt the user because that would make `setCredentials` asynchronous and that would cause the objs to be instantiated before this completes. But one step at a time.